features/user_can_create_account.feature

```diff
@@ -0,0 +1,19 @@
+Feature: User can create account
+    As a user
+    In order to use the service
+    I need to be able to create an account
+
+    Background:
+        Given I am on the landing page
+        And I click on "Sign up"
+        Then I should be on the Sign up page
```

Comment:
We never want to have any assertions in the background.

This test is also redundant. If we can fill out the form and successfully create an account, then we are on the sign up page. Please remove this test

Reply:
Yes, agree. Was added for debugging the little bomb.